CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+## [Unreleased]
+
+## [0.55.0] - 2018-04-03
+
+### Added
+- Script to update version automatically
+
+### Fixed
+- Fixed the parsing of entities for languages where tokens are not space separated
+- Japanese language label
+
+### Modified
+- Updated Rustling ontoloty to `0.16.4`
+
+
+[Unreleased]: https://github.com/snipsco/snips-nlu-ontology/compare/0.55.0...HEAD
+[0.55.0]: https://github.com/snipsco/snips-nlu-ontology/compare/0.54.0..0.55.0
```

Comment:
A dot is missing here `0.54.0..0.55.0` --> `0.54.0...0.55.0`

Reply:
Fixed